samples/charts/category-chart/tooltip-templates/Pages/CategoryChartTooltipTemplates.razor

```diff
@@ -0,0 +1,41 @@
+﻿@page "/CategoryChartTooltipTemplates"
```

Comment:
remove `samples/charts/category-chart/tooltip-templates` folder because this sample is not implemented in other platforms